spec/models/manageiq/providers/embedded_ansible/automation_manager/credential_spec.rb

```diff
@@ -1,5 +1,9 @@
 require 'support/ansible_shared/automation_manager/credential'
 
 describe ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ScmCredential do
-  it_behaves_like 'ansible credential', :provider_embedded_ansible
+  let(:ansible_manager) do
+    FactoryGirl.create(:provider_embedded_ansible, :with_authentication, :with_default_organization).managers.first
```

Comment:
I don't think it's reusable enough to justify adding a trait to the factory, this should work:
```ruby
FactoryGirl.create(:provider_embedded_ansible, :with_authentication, :default_organization => 1).managers.first
```